src/test/java/com/wikia/webdriver/common/logging/TestSuiteLogging.java

```diff
@@ -0,0 +1,42 @@
+package com.wikia.webdriver.common.logging;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by robertchan on 4/13/15.
+ */
+public class TestSuiteLogging {
+
+  private String testName;
+  private String className;
+  private String date;
+  private String time;
+  private String browser;
+  private String OS;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this field "OS" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00116)